mmcv/fileio/handlers/base.py

```diff
@@ -3,6 +3,7 @@
 
 
 class BaseFileHandler(metaclass=ABCMeta):
+    is_str_like_obj = True
```

Comment:
may explain what is this attribute means in comments, it is not so straightforward for now.

Reply:
add a comment for the flag